src/core.js

```diff
@@ -96,7 +96,9 @@ export default class Editable {
    * @chainable
    */
   add (target) {
-    this.enable($(target))
+    const targets = domArray(target, this.win.document)
```

Comment:
`this.enable(target)` would also work here as `domArray()` is called within `enable` again.